packages/plugin-odoo/src/Odoo.ts

```diff
@@ -51,15 +52,51 @@ export class Odoo<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
                 },
             },
             {
-                id: 'OdooColorPickerButton',
-                async render(): Promise<ActionableNode[]> {
+                id: 'OdooTextColorButton',
+                async render(): Promise<DividerNode[]> {
                     const button = new ActionableNode({
-                        name: 'colorpicker',
-                        label: 'Color picker',
-                        commandId: 'openColorPicker',
-                        modifiers: [new Attributes({ class: 'fa fa-eyedropper fa-fw' })],
+                        name: 'textcolorpicker',
+                        label: 'Text Color picker',
+                        commandId: 'openTextColorPicker',
+                        modifiers: [
+                            new Attributes({ class: 'fa fa-font fa-fw dropdown-toggle' }),
+                            new Attributes({ 'data-toggle': 'dropdown' }),
+                        ],
                     });
-                    return [button];
+                    const dropdownContent = new DividerNode();
+                    dropdownContent.modifiers.append(new Attributes({ class: 'dropdown-menu' }));
+                    const dropdownContainer = new DividerNode();
+                    dropdownContainer.modifiers.append(
+                        new Attributes({ class: 'dropdown jw-dropdown jw-dropdown-textcolor' }),
+                    );
+                    dropdownContainer.append(button);
+                    dropdownContainer.append(dropdownContent);
+                    return [dropdownContainer];
+                },
```

Comment:
There is a function you can use that converts html, I think it is `parseElement` so you could write something like:
```
parseElement('<div class="dropdown-menu"><div class="dropdown jw-dropdown ...`);
```
Just saying it exists in case you did not know, not necessarily that you ought to use it.

Reply:
Nice to know. thanks.